backend/src/main/resources/liquibase/db-changelog-1.0.xml

```diff
@@ -689,7 +689,7 @@
                 <constraints primaryKey="true" nullable="false"/>
             </column>
             <column name="migration_name" type="varchar(255)"/>
-            <column name="status" type="tinyint(1)"/>
+            <column name="status" type="tinyint"/>
```

Comment:
We shouldn't change existing migrations, but since we're switching away from Liquibase it doesn't really matter at this point.